modules/controller/lib/configure_controller.js

```diff
@@ -0,0 +1,44 @@
+const _ = require('lodash');
+
+const payloads = {
+  create: require('./payloads/create'),
+  read: require('./payloads/read'),
+  readRelation: require('./payloads/read'),
+  update: require('./payloads/update'),
+  destroy: require('./payloads/destroy')
+};
+
+const sourceInterface = {
+  create: require('./source/create'),
+  read: require('./source/read'),
+  readRelation: require('./source/read_relation'),
+  update: require('./source/update'),
+  destroy: require('./source/destroy')
+};
+
+const responder = require('./responder');
+
+module.exports = function (method, source, opts) {
+  if (!method) {
+    throw new Error('No method specified.');
+  }
+  if (!source) {
+    throw new Error('No source specified.');
+  }
```

Comment:
These conditionals aren't reachable given how this function is called from modules/controller/index.js. May still want to leave them for future security.